src/Sulu/Bundle/MediaBundle/Resources/js/components/ImageFocusPoint/ImageFocusPointCell.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {Icon} from 'sulu-admin-bundle/components';
+import type {Point} from './types';
+import imageFocusPointCellStyles from './imageFocusPointCell.scss';
+
+const ARROW_UP_ICON = 'arrow-up';
+
+type Props = {
+    size: number,
+    value: Point,
+    active: boolean,
+    onClick?: (value: Point) => void,
+    arrowDirection?: '-90deg' | '-45deg' | '0deg' | '45deg' | '90deg' | '125deg' | '180deg' | '225deg',
```

Comment:
Sorry, I wasn't clear enough about that... I would have been going for something like `up-left`, `up`, `up-right`, `left`, ... You get the idea. Don't you think that would be nicer? If I read deg I always have to think about what does negative mean, where does it start and so on 😕